docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,196 @@
+Run the command `edgectl install`. It will print something similar to the following:
 
-5. To access the Edge Policy Console going forward, you can access it using one of the following options:
-* `edgectl login -n <namespace> <AES_host>` or
-* `https://{{AES_URL}}/edge_stack/admin`
+```bash
+$ edgectl install
+-> Installing the Ambassador Edge Stack 1.0.
+-> Automatically configuring TLS.
+-> Cluster is not publicly accessible. Please ensure your cluster is publicly accessible if you would like to use automatic TLS.
 
-## Configure TLS Termination and Automatic HTTPS
+Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit http://192.168.64.2:31334 to access your Edge Stack installation and for additional configuration.
+```
 
-**The Ambassador Edge Stack enables TLS termination by default using a self-signed certificate. See the [Host CRD](/reference/host-crd) for more information about disabling TLS.** If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. If you do not have the ability to update your DNS, skip to the next section, "Create a Mapping."
+Your browser will automatically open to your newly provisioned URL.
 
-1. Update your DNS so that your domain points to the IP address for your cluster. 
+### Install in CI
 
-2. In the Edge Policy Console, create a `Host` resource:
-   * On the "Hosts" tab, click the **Add** button on the right.
-   * Enter your hostname (domain) in the hostname field.
-   * Check the "Use ACME to manage TLS" box.
-   * Review the Terms of Service and check the box that you agree to the Terms of Service.
-   * Enter the email address to be associated with your TLS certificate.
-   * Click the **Save** button.
-  
-  You'll see the newly created `Host` resource appear in the UI with a status of "Pending." This will change to "Ready" once the certificate is fully provisioned. If you receive an error that your hostname does not qualify for ACME management, you can still configure TLS following [these instructions](../../reference/core/tls) or by reviewing configuration in the [Host CRD](/reference/host-crd).
+Installing the Ambassador Edge Stack in your Continuous Integration tool requires additional configuration. This method does not automatically configure TLS termination for you, and therefore not a domain name. However, these can still be achieved with the Host CRD.
 
-3. Once the Host is ready, navigate to `https://<hostname>` in your browser. Note that the certificate warning has gone away. Additionally, the Ambassador Edge Stack automatically will redirect HTTP connections to HTTPS.
+To install the Ambassador Edge Stack in CI, run the following command:
+
+```bash
+edgectl install --ci
+```
+
+You must then configure your own TLS certificate using the [Host CRD](/reference/host-crd). Read more about [Edge Control in CI](/reference/edge-control-in-ci).
```

Comment:
That "Edge Control in CI" doc is about something completely different -- yay for having one tool that does _All The Things!_ But yeah, here we are talking about install Ambassador Edge Stack in CI; that doc is talking about using Edge Control for local development-style stuff in CI.